java/Glacier2/greeter/build.gradle.kts

```diff
@@ -4,6 +4,14 @@ plugins {
     // Register the Slice tools plugin without applying it here.
     // The plugin will be applied to all subprojects in the `subprojects` block below.
     id("com.zeroc.ice.slice-tools") version "3.8.0-+" apply false
+    id("checkstyle")
+    id("org.openrewrite.rewrite") version "7.3.0"
+}
+
+allprojects {
+    repositories {
+        mavenCentral()
+    }
 }
```

Comment:
Can we move this down, and don't use allprojects:

```
repositories {
    // For rewrote libraries
    mavenCentral()
}

rewrite {
```

The subprojects already has a repositories block. Or we can move the one from subprojects up here to allprojects. But we shouldn't have both `mavenCentral` in allprojects and in subprojects.